src/cascadia/TerminalApp/App.cpp

```diff
@@ -56,6 +56,16 @@ namespace winrt::TerminalApp::implementation
         // registered?" when it definitely is.
     }
 
+    float App::GetNonClientAreaDragBarWidth() const noexcept
+    {
+        if (_settings->GlobalSettings().GetShowTabsInTitlebar() == false)
+        {
+            return 0.0;
+        }
+
+        return NON_CLIENT_DRAGBAR_WIDTH;

```

Comment:
Should this return a variable amount, based on the width of the window?

Reply:
Yes, and no, this is a minimum width. We still need to do work on TabItem and the split button to dynamically move it (and adject this width in NonClientWindow via some notification)